recipes-graphics/weston/weston-conf.bbappend

```diff
@@ -20,4 +20,12 @@ do_install_raspberrypi3-64() {
 [core]
 backend=fbdev-backend.so
 EOF
-}
\ No newline at end of file
+}
+
+do_install_beaglebone() {
+        mkdir -p ${D}/${sysconfdir}/xdg/weston
+        cat << EOF > ${D}/${sysconfdir}/xdg/weston/weston.ini
+[core]
+backend=fbdev-backend.so
+EOF
+}
```

Comment:
Thanks a lot!
If you have time, could you make a common function for the use of fbdev-backend? 
(If not, I will just merge current this PR.)

```
do_install_fbdev_backend () {
        mkdir -p ${D}/${sysconfdir}/xdg/weston
        cat << EOF > ${D}/${sysconfdir}/xdg/weston/weston.ini
[core]
backend=fbdev-backend.so
EOF
}

do_install_qemuarm () {
        do_install_fbdev_backend
}
...
do_install_beaglebone () {
        do_install_fbdev_backend
}
```

Reply:
Yes! I think it's not a good implementation, too. 
So I rewrite weston-conf.bbappend, please look 0a4a6a0.

Reply:
> do_install_fbdev_backend () {
...(snip)...
> do_install_beaglebone () {
>         do_install_fbdev_backend
> }

I'm sorry to ignore your sample code. I just realized it. 
Thank you for your suggestions. 

However, I feel the your code is a bit messy because it needs to add or fix (when user want to use without fbdev) do_install() for each board.
(I know it's same way of thinking of poky-warrior)
my patch override do_install() in poky layer, however it has compatible.

Reply:
Your code is better than mine. Thank you.
However, the DEFAULTBACKEND should be "fbdev" instead of empty string, shouldn't it?

Reply:
>  I think that default value of the DEFAULTBACKEND should be "fbdev" instead of empty string so that we can skip the following check.

I think do_install() should be skipped by default (in other words DEAFULTBACKEND should be set empty by default)
because do_install() in poky's weston-conf.bb was written to do nothing by default[1] other than qemux86/qemux86-64.
Otherwise my patch may have a negative effect when built on other machines.

[1] https://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/recipes-graphics/wayland/weston-conf.bb?h=warrior

Reply:
Got it. I will merge this PR.
Thank you!